.pre-commit-config.yaml

```diff
@@ -31,8 +31,6 @@ repos:
           - '--args=--only=terraform_required_providers'
           - '--args=--only=terraform_standard_module_structure'
           - '--args=--only=terraform_workspace_remote'
-      - id: terraform_validate
-        exclude: docs
```

Comment:
@bryantbiggs Is this step used in any of the pre-commit GitHub workflows? What is your recommendation here?

Reply:
if your CI is already running a `terraform plan` on each pattern, then this could be considered redundant

I also don't see pre-commit as a CI check which is why I guess a lot of these white space/noise changes are occurring on subsequent PRs